test/jdk/java/awt/print/PrinterJob/PrintImage.java

```diff
@@ -21,276 +21,167 @@
  * questions.
  */
 
-/**
+import java.awt.BorderLayout;
+import java.awt.Canvas;
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Frame;
+import java.awt.Graphics;
+import java.awt.Menu;
+import java.awt.MenuBar;
+import java.awt.MenuItem;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
+
+import jtreg.SkippedException;
+
+/*
  * @test %I %W
  * @bug 4298489
  * @summary Confirm that output is same as screen.
  * @key printer
+ * @library /test/lib /java/awt/regtesthelpers
+ * @build PassFailJFrame jtreg.SkippedException
  * @run main/manual PrintImage
  */
-import java.awt.*;
-import java.awt.print.*;
-import java.awt.event.*;
-
 public class PrintImage extends Frame implements ActionListener {
 
-        private PrintImageCanvas                printImageCanvas;
+    private PrintImageCanvas printImageCanvas;
 
-        private MenuItem        print1Menu = new MenuItem("PrintTest1");
-        private MenuItem        print2Menu = new MenuItem("PrintTest2");
-        private MenuItem        exitMenu = new MenuItem("Exit");
+    private final MenuItem print1Menu = new MenuItem("PrintTest1");
+    private final MenuItem print2Menu = new MenuItem("PrintTest2");
+    private final MenuItem exitMenu = new MenuItem("Exit");
 
-        public static void main(String[] argv) {
-        String[] instructions =
-           { "You must have a printer available to perform this test,",
-             "prefererably Canon LaserShot A309GII.",
-             "Printing must be done in Win 98 Japanese 2nd Edition.",
-             "",
-             "Passing test : Output of text image for PrintTest1 and PrintTest2 should be same as that on the screen.",
-           };
+    private static final String INSTRUCTIONS =
+            "You must have a printer available to perform this test,\n" +
+            "prefererably Canon LaserShot A309GII.\n" +
+            "Printing must be done in Win 98 Japanese 2nd Edition.\n" +
```

Comment:
I don't think it's possible at all. Nor is it relevant at the moment.

You should look at the original bug. If the test is still relevant, please update the description. If not, let's remove it.